_posts/2019-04-23-gsod2019.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: Google Season of Docs 2019 Application
+---
+
+<p>
+<img
+src="https://developers.google.com/season-of-docs/images/SeasonofDocs_Icon_Grey_300ppi_trimmed.png"
+title="Google Season of Docs 2019" alt="Google Season of Docs 2019"
+style="float: right; height: 8em; " />
+</p>
+
+MDAnalysis is applying for [Google Season of Docs 2019][gsod]. We would
+love to have a technical writer work with us to improve our
+documentation.
+
+In this program, Google is sponsoring a professional technical writer
+to work with an open source project. The open source project gets to
+work with a professional in the area of documentation --- something
+that almost all projects struggle with and want to improve --- and
+the writer gets experience working with an open
+source project, e.g., being embedded in the developer community and
+using tools such as automatic doc generation and continuous
+integration services.
+
+Because MDAnalysis is a project *written by scientists for
+scientists*, a writer with MDAnalysis would not only **learn about the
+technical aspects** of creating docs for open source code (such as
+*documentation generation with Sphinx*, *continuous integration-based
+workflows* and *distributed development with git* via our GitHub source code
+repository) but also about the science that MDAnalysis
+enables. Our developers have published many scientific publications
+that used our library or described innovative new methods that were
+implemented in MDAnalysis. A writer would interact with these
+accomplished scientists and **learn about the science**: *How
+molecules (especially biomolecules such as proteins and DNA) are
+simulated* and *how scientists analyze such simulations in order to
+better understand the world at the microscopic scale*.
+
+
+
+# What is MDAnalysis?
+
+MDAnalysis is a **[Python][] library** for the analysis of particle-based
```

Comment:
maybe expand and make this sentence more accessible?  "these particles represent the motion of atoms which make up our bodies and the natural world"?

Reply:
MDAnalysis is a **[Python][] library** for the analysis of computer
simulations of many-body systems at the molecular scale.